### DISCUSSION TITLE
Dark Mode, como usar?

### DISCUSSION THREAD

**Discussion by Feeh03114** [po-ui/po-angular#1094] [Tire suas dúvidas]
Como eu faço para colocar o dark mode na po ui?

**Comment by idxtecnologia** [comment]
Também gostaria de saber se existe possibilidade de alternar entre tema escuro e tema claro?

**Comment by rafaellmarques** [comment]
Boa tarde @Feeh03114 e @idxtecnologia , todos bem?

Oficialmente não temos uma maneira de alternar os temas entre claro e escuro.
Estamos trabalhando com o time de UX para no futuro implementarmos essa possibilidade.

Att.
